### PR TITLE
Issue#63 frontend continue mobile style

### PR DESCRIPTION
**Pull request by mariagluck**
so here is more of the styling of the mobile. review and merge but do not close issue and branch bitte :))

### CODE REVIEW

**Comment by Mxx1029** [comment]
ok, will merge, but not delete the branch. as you didn't link the issue, the PR will not close it anyway :)

File: package.json [hunk 1]
```diff
@@ -48,5 +48,5 @@
   "devDependencies": {
     "node-sass": "^6.0.1"
   },
-  "proxy": "http://localhost:3001"
+  "proxy": "http://localhost:3000"
 }
```
Comment (Mxx1029) [inline package.json:52]: can we at some point change this to port 9012 and then shorten all axis urls to /users or /events or whatever?

**Review by Mxx1029 — APPROVED** [review]
Very nice!! Getting better and better:
1 thing: there's an empty file called FETCH_HEAD in the repo, that I'm not sure why it's there...?

File: package.json (hunk 1 above)
Comment (mariagluck) [reply package.json:52]: Yes sure! I inherited this in my last pull from origin...I had previously just the 9012 backend port, i can change it again

**Comment by mariagluck** [comment]
> Very nice!! Getting better and better: 1 thing: there's an empty file called FETCH_HEAD in the repo, that I'm not sure why it's there...?

ohoo, no idea...I will look asap what is that funky thing

**Pull request merged**